Mathlib/Data/Real/Pointwise.lean

```diff
@@ -22,7 +23,7 @@ This is true more generally for conditionally complete linear order whose defaul
 don't have those yet.
 -/
 
-assert_not_exists Finset
+--assert_not_exists Finset
```

Comment:
Hmm, this should either be removed, or we need a new file.

@kim-em opinions?

Reply:
Right, I left it as a comment because I did not know which option would be preferred.

Reply:
I put the `assert_not_exists` back after following @YaelDillies' suggestion.